src/ui/components/DelegateVoting.tsx

```diff
@@ -96,9 +96,11 @@ export function DelegateVoting({fromAccount, onClose}: DelegateVotingProps) {
           method: 'democracy.delegate',
           params: [delegatedAccount, conviction.value, bnToHex(delegateAmountBn)],
         },
+      }).then(() => {
+        refetchConvictions();
       });
     }
-  }, [delegatedAccount, conviction, delegateAmountBn, fromAccount?.address, startTx]);
+  }, [delegatedAccount, conviction, delegateAmountBn, fromAccount?.address, startTx, refetchConvictions]);
```

Comment:
voting delegation doesn't affect conviction in any way. So no need to refetch conviction data after delegation.

Reply:
This component does't have conviction hook.